percona-xtradb-cluster-5.7-backup/Dockerfile

```diff
@@ -32,10 +32,10 @@ LABEL org.opencontainers.image.description="Percona XtraDB Cluster is a high ava
 helps enterprises avoid downtime and outages and meet expected customer experience."
 LABEL org.opencontainers.image.license="GPL"
 
-ENV PXB_VERSION 2.4.22-1
+ENV PXB_VERSION 2.4.23-1
 ENV PXC_VERSION 5.7.33-31.49.1
-ENV KUBECTL_VERSION=v1.16.15
-ENV KUBECTL_SHA256SUM=e8913069293156ddf55f243814a22d2384fc18b165efb6200606fdeaad146605
+ENV KUBECTL_VERSION=v1.19.12
```

Comment:
we need version which works with all supported versions
e.g. 1.19 supports 1.18 and 1.20

Reply:
@delgod yeah, but we support from 1.16 to 1.21 - should I just leave this at 1.16?